curtsies/events.py

```diff
@@ -299,42 +360,47 @@ def try_keys():
 
     def ask_what_they_pressed(seq, Normal):
         # type: (bytes, Termmode) -> None
-        print('Unidentified character sequence!')
+        print("Unidentified character sequence!")
         with Normal:
             while True:
                 r = raw_input("type 'ok' to prove you're not pounding keys ")
-                if r.lower().strip() == 'ok':
+                if r.lower().strip() == "ok":
                     break
         while True:
-            print('Press the key that produced %r again please' % (seq,))
+            print("Press the key that produced %r again please" % (seq,))
             retry = os.read(sys.stdin.fileno(), 1000)
             if seq == retry:
                 break
             print("nope, that wasn't it")
         with Normal:
-            name = raw_input('Describe in English what key you pressed: ')
-            f = open('keylog.txt', 'a')
+            name = raw_input("Describe in English what key you pressed: ")
+            f = open("keylog.txt", "a")
             f.write("%r is called %s\n" % (seq, name))
             f.close()
-            print('Thanks! Please open an issue at https://github.com/bpython/curtsies/issues')
-            print('or email thomasballinger@gmail.com. Include this terminal history or keylog.txt.')
-            print('You can keep pressing keys')
+            print(
```

Comment:
I don't like this at all, but fair. I wonder about setting the line length to 100 instead of 88